docs/topics/modules/using_terminal_tab.adoc

```diff
@@ -1,17 +1,15 @@
 [id="using_terminal_tab"]
 = Using the terminal tab
 
-Your Che workspace displays a terminal tab at the bottom of the screen. This terminal is for your private Linux container (i.e. your workspace). You can use Unix commands in the terminal tab as follows: 
+Your Che workspace displays a terminal tab at the bottom of the screen. You can use this terminal to run commands in your private Linux container (i.e. your workspace) as follows:
```

Comment:
Not a part of this PR, but afaiu from the original doc, the terminal itself is the private linux container or workspace.
How about, You can use this terminal as your private Linux container or your workspace to run commands as follows.

i.e.>that is

Reply:
I'm not sure that a terminal itself can be the workspace or container. Surely it's just a tool used to access the container or workspace and run commands? If there's a confirmation from the relevant devs, I'll change this. Todd's document isn't particularly clear on this, apparently, since I read it as "this" = what you are accessing via the terminal, not the terminal itself when it says "This is your private workspace".